packages/retail-ui-extensions/src/extension-api/navigation-api/navigation-api.ts

```diff
@@ -1,31 +1,26 @@
+import type {Modal} from '../types';
+
 export interface NavigationApiContent {
-  /** The name of the current screen on the stack.
-   * This property can be used to evaluate which component you want to mount.
+  /** Contains the information about the currently presented modal */
+  modal: Modal;
```

Comment:
Should we call this something like `currentModal` instead?

Reply:
That would make sense, yeah, renamed it.